fvm/env.go

```diff
@@ -301,6 +371,80 @@ func (env *commonEnv) GetAccountContractNames(address runtime.Address) ([]string
 	return env.accounts.GetContractNames(a)
 }
 
+func (env *commonEnv) ResolveLocation(
+	identifiers []runtime.Identifier,
+	location runtime.Location,
+) ([]runtime.ResolvedLocation, error) {
+	defer env.StartExtensiveTracingSpanFromRoot(trace.FVMEnvResolveLocation).End()
+
+	err := env.Meter(meter.ComputationKindResolveLocation, 1)
+	if err != nil {
+		return nil, fmt.Errorf("resolve location failed: %w", err)
+	}
+
+	addressLocation, isAddress := location.(common.AddressLocation)
+
+	// if the location is not an address location, e.g. an identifier location
+	// (`import Crypto`), then return a single resolved location which declares
+	// all identifiers.
+	if !isAddress {
+		return []runtime.ResolvedLocation{
+			{
+				Location:    location,
+				Identifiers: identifiers,
+			},
+		}, nil
+	}
+
+	// if the location is an address,
+	// and no specific identifiers where requested in the import statement,
+	// then fetch all identifiers at this address
+	if len(identifiers) == 0 {
+		address := flow.Address(addressLocation.Address)
+
+		err := env.accounts.CheckAccountNotFrozen(address)
+		if err != nil {
+			return nil, fmt.Errorf("resolving location failed: %w", err)
```

Comment:
```suggestion
			return nil, fmt.Errorf("checking account not frozen failed: %w", err)
```

Reply:
updated this error string since it's less clear.